### DISCUSSION TITLE
Video ADempiere 394

### DISCUSSION THREAD

**Discussion by AlbertoAbudinen** [adempiere/adempiere#4123] [General]
Estamos a la espera del YouTube oficial de ADempiere para continuar

El objetivo de esta discusión es generar un listado de videos , inicialmente en YouTube, y luego cuando tengamos las cuentas de ADempiere oficial subir los links a la web y/o github o similar.

El temario de los videos a realizar serán (esto es un trabajo en transito ) :

login de usuario
introducción a Adempiere y ventanas
Maestros
flujo de compra estándar
flujo de venta estándar

vamos a ir respondiendo este misma discusión a medida que se vayan generando los manuales y  videos. y pueden aportar con cualquier link aquí mismo también.

**Comment by yamelsenih** [comment]
Hay una cuenta oficial? Propongo simplemente crear una nueva y ya. No toma más de media hora y no se necesita un correo corporativo o dominio corporativo para eso, simplemente una cuenta de Gmail.

Nota: la cuenta debería tener como mínimo dos o formas de recuperación y que esté en poder de más de dos personas y no solo una.

Saludos

**Reply by dibya-the-coder** [reply]
write in English

**Comment by marcalwestf** [comment]
En casos como el _Flujo de Compras_ donde están involucradas varias ventanas y/o pestañas,  en nuestras instalaciones, utilizamos los flujos de ADempiere para que los usuarios tengan una visión general del flujo.

Ejemplo:
![Selection_168](https://github.com/adempiere/adempiere/assets/1789408/ee991e0f-efff-41ad-8d41-264c2b5f34af)

**Reply by dibya-the-coder** [reply]
🙄

**Comment by AlbertoAbudinen** [comment]
oka vamos a incluir flujos , tal vez sea mas claro si los hacemos en BPMN .

**Comment by juddm** [comment]
Nice initiative - I would be happy to do some of the finance topics
